frontend/src/pages/teacher/ProjectViewTeacher.tsx

```diff
@@ -37,6 +40,24 @@ export default function ProjectViewTeacher() {
         groupProject: project_data.project_max_students > 1,
     }
 
+    async function downloadAllSubmissions() {
+        let url = ''
+        if (DEBUG) {
+            url = 'http://localhost:8000'
+        }
+        const response = await fetch(
+            `${url}/api/projects/${project.projectId}/submissions`,
+            {headers: {'Authorization': 'Bearer ' + localStorage.getItem('token')}}
+        )
```

Comment:
Kan je ApiFetch niet gebruiken?
Je kan ook een options meegeven aan apiFetch wat letterlijk u 2de arg is dat je aan fetch meegeeft hier.